packages/build/tests/core/tests.js

```diff
@@ -522,6 +522,21 @@ test.serial('functions can have a config with different parameters passed to zip
   t.snapshot(normalizeOutput(output))
 })
 
+test.serial('internalSrcFolder is passed to zip-it-and-ship-it and helps prefill the generator field', async (t) => {
```

Comment:
I don't think Netlify Build knows anything about the functions manifest or the generator field?

```suggestion
test.serial('internalSrcFolder is passed to zip-it-and-ship-it', async (t) => {
```

Reply:
This is more as a documentation of why we are passing it and what we are testing in this test :)